File: backend/SmakApi/Models/DTOs/ReviewDto.cs
namespace SmakApi.Models.DTOs;

public class ReviewDto
{
    public string Author { get; set; } = null!;
    public Guid AuthorId { get; set; }
    public string? AvatarUrl { get; set; }
    public int Rating { get; set; }
    public string? Comment { get; set; }
    public DateTime CreatedAt { get; set; }
}

File: mobile/app/src/main/java/com/itstep/smakmobile/dto/ReviewDto.java
package com.itstep.smakmobile.dto;

import java.util.Date;
import java.util.UUID;

public class ReviewDto {
    public UUID id;
    public String author;
    public UUID authorId;
    public String avatarUrl;
    public int rating;
    public String comment;
    public Date createdAt;

    public UUID getId() {
        return id;
    }

    public String getAuthor() {
        return author;
    }

    public UUID getAuthorId() {
        return authorId;
    }

    public String getAvatarUrl() {
        return avatarUrl;
    }

    public int getRating() {
        return rating;
    }

    public String getComment() {
        return comment;
    }

    public Date getCreatedAt() {
        return createdAt;
    }
}
